localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -0,0 +1,45 @@
+﻿l_english:
+#Journal Entries
+ #je_second_revolt_of_bosnian_beys:0 "The Revolt of Bosniak Beys"
```

Comment:
Any reason to keep these commented out translation not related to Venice here?

Reply:
When I uploaded it I was starting to work on events and the journal system and needed a reference. In my next push request it will be gone and replaced with actual events and localization.